packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -748,6 +749,10 @@ describe("normalizeAndValidateConfig()", () => {
           cwd: "CWD",
           watch_dir: "WATCH_DIR",
         },
+        define: {
+          DEF1: "DEFINE_1",
+          DEF2: "DEFINE_2",
```

Comment:
Can you add a test for when the user does not provide a string?

Reply:
done and done, expanded it a fair bit